### PR TITLE
#630 Docker image jeroenwillemsen/wrongsecrets:1.5.14-no-vault hangs …

### PR DESCRIPTION
**Pull request by MarcinNowak-codes**
…on Colima container runtimes on macOS with M1

Thank you for submitting a pull request to the WrongSecrets app!

What kind of changes does this PR include?

-   [ ] Fixes or refactors
-   [ ] A new challenge
-   [x] Additional documentation for #630
-   [ ] Something else

Checklist:

-   [x] All the contributions made are solely the work of me and my co-authors
-   [x] I tested the changes in this PR (if applicable)
-   [ ] I added unit tests to ensure my change works (when change in Java or on front-end code)
-   [x] The PR passes pre-commit hooks and automated tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -426,6 +426,37 @@ And then at [http://localhost:3000](http://localhost:3000).
 
 Note: be careful with trying to deploy the `jeroenwillemsen/wrongsecrets-desktop` container to Heroku ;-).
 
+## Docker on macOS with M1 and Colima
```
Comment (commjoen) [inline README.md:429]: can you please mark this as "experimental support" ?

**Comment by commjoen** [comment]
When you have approved https://github.com/OWASP/wrongsecrets/pull/651 , then you can refer to the actions for alternative docker runtimes like colima and podman :).

**Comment by commjoen** [comment]
@MarcinNowak-codes do you have any update on this :-)?

**Comment by commjoen** [comment]
Hi @MarcinNowak-codes , do you have any update on this? Will have to close the pr this week due to inactivity.

**Comment by commjoen** [comment]
Welcome back sir!

File: README.md (hunk 1 above)
Comment (MarcinNowak-codes) [reply README.md]: Marked

**Comment by MarcinNowak-codes** [comment]
I've just checked again and simplify how to run with different Docker context.

**Review by commjoen — APPROVED** [review]
Thanks!

**Pull request merged**